app/routes/models/search/resultsModel.ts

```diff
@@ -203,13 +220,15 @@ export const createModel = async (query: any, error?: any) => {
     counties: typeof query.counties === 'string' ? [query.counties]: query.counties
   }
 
-  const { matches, total } = await performSearch(searchString, requestedPage, filterBy, sortBy)
-  
+  const { results, total, filterOptions } = await performSearch(searchString, requestedPage, filterBy, sortBy)
+  const amounts = getMatchingStaticAmounts(filterOptions?.amounts)
```

Comment:
does it make sense to rename this amount to specific , something like filterAmounts?

Reply:
Its called amounts because it gets passed as 'amounts' to the getFilters function below. Happy to merge the whole line to 230 if it improves readability